perun/utils/common/common_kit.py

```diff
@@ -11,6 +14,7 @@
 import signal
 
 # Third-Party Imports
+import array
```

Comment:
I believe `array` is part of the Python standard library? https://docs.python.org/3/library/array.html